docs/introduction.md

```diff
@@ -69,10 +69,40 @@ For more information, see the [API documentation](api-overview.md).
 ### Cloud optimizations
 To improve the performance of applications that run in containers, try setting the following tuning options:
 
-- Use a shared classes cache (`-Xshareclasses -XX:SharedCacheHardLimit=200m -Xscmx60m`) with Ahead-Of-Time (AOT) compilation to improve your startup time. For persistence, store the cache in a volume that you map to your container. For more information, see [Inroduction to class data sharing](shrc.md) and [AOT Compiler](aot.md).
+- Use a shared classes cache (`-Xshareclasses -XX:SharedCacheHardLimit=200m -Xscmx60m`) with Ahead-Of-Time (AOT) compilation to improve your startup time. For persistence, store the cache in a volume that you map to your container. For more information, see [Introduction to class data sharing](shrc.md) and [AOT Compiler](aot.md).
 
 - Use the [-Xtune:virtualized](xtunevirtualized.md) option, which configures OpenJ9 for typical cloud deployments where VM guests are provisioned with a small number of virtual CPUs to maximize the number of applications that can be run. When enabled, OpenJ9 adapts its internal processes to reduce the amount of CPU consumed and trim down the memory footprint. These changes come at the expense of only a small loss in throughput.
 
+- Provide access to the `/proc` file system for container detection on Linux systems since the detection code requires access to the `/proc/1/cgroup` and `/proc/1/sched` files. If you mount the `/proc` file system with the `hidepid=2` option on Linux systems and you do not have root privileges, the VM started by you cannot access the `/proc` file system and the container detection fails. Even though the container detection fails, the VM does start with a warning message. For example:
```

Comment:
We can add `®` with Linux for consistency.
```suggestion
- Provide access to the `/proc` file system for container detection on Linux® systems since the detection code requires access to the `/proc/1/cgroup` and `/proc/1/sched` files. If you mount the `/proc` file system with the `hidepid=2` option on Linux® systems and you do not have root privileges, the VM started by you cannot access the `/proc` file system and the container detection fails. Even though the container detection fails, the VM does start with a warning message. For example:
```

Reply:
`you do not have root privileges`: and the VM doesn't have root privileges ...

Reply:
> We can add `®` with Linux for consistency.

Only the first instance of Linux in this topic needs the registration mark. We don't give it for any subsequent instances.